[Bug]: Node operations failed on baremetal ai

### Description
Node operation is not working with these config values `platform: 'baremetal', deployment_type: 'ai'`
```
>           nodes_multicluster[primary_cluster_index].stop_nodes(primary_cluster_nodes)

tests/functional/disaster-recovery/regional-dr/test_failover.py:121: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/platform_nodes.py:2149: in stop_nodes
    self.baremetal.stop_baremetal_machines(nodes, force=force)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.utility.baremetal.BAREMETAL object at 0x7fe666de30d0>
baremetal_machine = [<ocs_ci.ocs.resources.ocs.OCS object at 0x7fe6653184d0>, <ocs_ci.ocs.resources.ocs.OCS object at 0x7fe665b1b290>, <oc...850>, <ocs_ci.ocs.resources.ocs.OCS object at 0x7fe666f23890>, <ocs_ci.ocs.resources.ocs.OCS object at 0x7fe666f20510>]
force = True

    def stop_baremetal_machines(self, baremetal_machine, force=True):
        """
        Stop Baremetal Machines
    
        Args:
            baremetal_machine (list): BM objects
            force (bool): True for BM ungraceful power off, False for
                graceful BM shutdown
    
        Raises:
            UnexpectedBehaviour: If baremetal machine is still up
    
        """
        for node in baremetal_machine:
            if force:
                if self.srv_details[node.name]:
                    ipmi_ctx = self.get_ipmi_ctx(
>                       host=self.srv_details[node.name]["mgmt_console"],
                        user=self.srv_details[node.name]["mgmt_username"],
                        password=self.srv_details[node.name]["mgmt_password"],
                    )
E                   KeyError: 'mgmt_console'

ocs_ci/utility/baremetal.py:99: KeyError
```
RDR tests failed due to this error. The platform is IBM Cloud Bare Metal.

### Steps to Reproduce
1. Run tests/functional/disaster-recovery/regional-dr/test_failover.py::TestFailover::test_failover[primary_down-cephfs-cli]
or any other tests that does node operations.

### Actual Behavior
KeyError: 'mgmt_console'
### Expected Behavior
Identify the correct platform class
### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)
Node operation tests, RDR
### Screenshots (if applicable)

### Environment
- Test Suite(s): rdr
- Platform(s): IBM Cloud Bare Metal
- Version(s): 4.21
- OS:

### Additional Context

--- a/ocs_ci/ocs/platform_nodes.py
+++ b/ocs_ci/ocs/platform_nodes.py
@@ -96,7 +96,7 @@ class PlatformNodesFactory:
             "AWS": AWSNodes,
             "vsphere": VMWareNodes,
             "aws": AWSNodes,
-            "baremetal": IBMCloudBMNodes,
+            "baremetal": BaremetalNodes,
             "azure": AZURENodes,
             "gcp": GCPNodes,
             "vsphere_lso": VMWareLSONodes,
@@ -111,6 +111,7 @@ class PlatformNodesFactory:
             "hci_baremetal": IBMCloudBMNodes,
             "kubevirt_vm": KubevirtVMNodes,
             "ibm_cloud_ipi": IBMCloudIPI,
+            "baremetal_ai": IBMCloudBMNodes,
         }
 
     def get_nodes_platform(self):
@@ -136,6 +137,11 @@ class PlatformNodesFactory:
         if config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM:
             if config.ENV_DATA["deployment_type"] == "ipi":
                 platform += "_ipi"
+
+        if config.ENV_DATA["platform"] == constants.BAREMETAL_PLATFORM:
+            if config.ENV_DATA["deployment_type"] == "ai":
+                platform += "_ai"
+
         return self.cls_map[platform]()
 
 

